### PR TITLE
implement post token UI

### PR DESCRIPTION
**Pull request by meynet-salesforce**
Creating this for Anny since she can't login to github right now

### CODE REVIEW

File: view/tokens/new.pug [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  All rights reserved.
+  Licensed under the BSD 3-Clause license.
+  For full license text, see LICENSE.txt file in the repo root or
+  https://opensource.org/licenses/BSD-3-Clause
+
+
+doctype html
+html
+  head
+    title Create new token
```
Comment (iamigo) [inline view/tokens/new.pug]: title New Token

File: view/tokens/app.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    if (error) {
+      toggleVisibility(errorInfo, true);
+      errorInfo.innerHTML = 'An unexpected error occurred';
+      toggleVisibility(successInfo, false);
+      toggleVisibility(tokenInfo, false);
+    } else {
+      toggleVisibility(successInfo, true);
+      toggleVisibility(tokenInfo, true);
+      successInfo.innerHTML = 'Token generated from ' + res.body.name +
+        '. Please save this token, you will not see this token again!';
```
Comment (iamigo) [inline view/tokens/app.js]: how about something like this:

```
<b>Copy and paste your new token somewhere safe--you will not be able to see it again!</b>

Token name: ________
Token value: __________________
```

File: view/tokens/new.pug [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      div.text-center.col-md-4.col-md-offset-4
+        h1 Create new token
+        br
+        div.local-form
+          form(role='form', name='loginform')
+            #errorInfo
+            #successInfo
+            #tokenInfo
+            .form-group
+              input.form-control(type='text', autofocus, name="name", maxlength='60' placeholder='Enter token name' required)
```
Comment (iamigo) [inline view/tokens/new.pug]: When you get back the token value, can we put a little more vertical space and maybe even a horizontal line between the new token that gets returned and the form fields where you enter a name to create yet another new token?

**Comment by annyhe** [comment]
Updated with changes. Attached screenshots.
screen shot 2016-12-13 at 8 41 26 am
screen shot 2016-12-13 at 8 41 05 am

File: view/tokens/tokens.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
+/**
```
Comment (iamigo) [inline view/tokens/tokens.css]: pls add copyright to file

**Comment by annyhe** [comment]
Comments @shriramshankar ?

**Comment by iamigo** [comment]
@shriramshankar waiting for review

**Comment by shriramshankar** [comment]
@iamigo left a comment yesterday. Ok, to merge if we do not want to take care of that.

**Pull request merged**